.prettierrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "bracketSpacing": false,
+  "singleQuote": true,
+  "printWidth": 95
+}
```

Comment:
Hey @barrymcgee, when did we decide on these overrides? We just the defaults in rbac and maas-ui.

Reply:
I agree. However, these were the compromises I had to make to get Prettier accepted onto the old GUI codebase. I'd happily use the defaults, especially as our other projects do - over to you @hatched?